### ISSUE TITLE
Failing to the parse the following

### ISSUE THREAD

**Issue by atkawa7** [nilswende/dbml-java#2]
Thank you for the library

```dbml
Table Organization {
  organization_id integer [pk]
  name varchar [not null]
  code varchar [unique, not null]
  created_at timestamp [default: `CURRENT_TIMESTAMP`]
  updated_at timestamp [default: `CURRENT_TIMESTAMP`]
}
```

```java
 final var o = DbmlParser.parse("Table Organization {\n" +
                "  organization_id integer [pk]\n" +
                "  name varchar [not null]\n" +
                "  code varchar [unique, not null]\n" +
                "  created_at timestamp [default: `CURRENT_TIMESTAMP`]\n" +
                "  updated_at timestamp [default: `CURRENT_TIMESTAMP`]\n" +
                "}\n");
       System.out.println(o);
```

**Comment by nilswende** [comment]
Thank you for using it, and thank you for the bug report. 

I am looking into it.

**Comment by atkawa7** [comment]
@nilswende It seems the issue is around default values. I am. I am not sure what the spec says about it.  When you add single quotes around it works fine. 


```dbml
Table SubDivision {
  sub_division_id integer [pk]
  code varchar [unique, not null]
  name varchar [not null]
  local_name varchar
  description text
  is_active boolean [default: true]
  longitude decimal
  latitude decimal
  created_at timestamp
  updated_at timestamp
}
```



```java
 final  var o = DbmlParser.parse("Table SubDivision {\n" +
                "  sub_division_id integer [pk]\n" +
                "  code varchar [unique, not null]\n" +
                "  name varchar [not null]\n" +
                "  local_name varchar\n" +
                "  description text\n" +
                "  is_active boolean [default: true]\n" +
                "  longitude decimal\n" +
                "  latitude decimal\n" +
                "  created_at timestamp\n" +
                "  updated_at timestamp\n" +
                "}");

System.out.println(o);
```

**Comment by atkawa7** [comment]
@nilswende I have created a fix in #3

**Comment by nilswende** [comment]
v1.1.0 should fix the problem.
